OE-only:
comment: Not building the manpage is just a hack.

diff -Nur aufs.vanilla/util/Makefile aufs/util/Makefile
--- aufs.vanilla/util/Makefile	2008-08-23 13:00:37.000000000 +0200
+++ aufs/util/Makefile	2008-08-23 15:40:26.000000000 +0200
@@ -1,25 +1,10 @@
 
 # $Id: Makefile,v 1.10 2008/05/26 04:02:40 sfjro Exp $
 
-all: aufs.5 aulchown
+all: aulchown
 	test -x ./mount.aufs || chmod a+x ./mount.aufs
 
-aufs.5: aufs.in.5 ./c2tmac
-	${RM} $@
-	./c2tmac > $@
-	awk '{ \
-		gsub(/\140[^\047]*\047/, "\\[oq]&\\[cq]"); \
-		gsub(/\\\[oq\]\140/, "\\[oq]"); \
-		gsub(/\047\\\[cq\]/, "\\[cq]"); \
-		gsub(/\047/, "\\[aq]"); \
-		print; \
-	}' aufs.in.5 >> $@
-	chmod a-w $@
-
 CFLAGS += -O2 -Wall ${EXTRA_CFLAGS} ${ccflags-y}
 
-.INTERMEDIATE: c2tmac.o c2tmac
-c2tmac.o: c2tmac.c ../include/linux/aufs_type.h
-
 clean:
-	${RM} c2tmac aulchown aufs.5 *~ .*~
+	${RM} aulchown *~ .*~
